indico/modules/events/editing/models/editable.py

```diff
@@ -34,7 +34,8 @@ def editor_permission(self):
 class EditableState(RichIntEnum):
     __titles__ = [None, pgettext('Editable', 'New'), pgettext('Editable', 'Ready for Review'),
                   pgettext('Editable', 'Needs Confirmation'), pgettext('Editable', 'Needs Changes'),
-                  pgettext('Editable', 'Accepted'), pgettext('Editable', 'Rejected')]
+                  pgettext('Editable', 'Accepted'), pgettext('Editable', 'Accepted by Submitter'),
+                  pgettext('Editable', 'Rejected')]
     __css_classes__ = [None, 'highlight', 'ready', 'editing-make-changes', 'editing-request-changes',
                        'success', 'editing-rejected']
```

Comment:
You need to add your new title to the end (since it's index 7), and also add a CSS class. Right now things break since it tries to read `__css_classes__[7]` which does not exist.